lib/msal-browser/src/interaction_handler/SilentHandler.ts

```diff
@@ -35,22 +35,22 @@ export class SilentHandler extends InteractionHandler {
 
     /**
      * Monitors an iframe content window until it loads a url with a known hash, or hits a specified timeout.
-     * @param iframeContentWindow 
+     * @param iframe 
      * @param timeout 
-     * @param urlNavigate 
      */
-    monitorFrameForHash(iframe: HTMLIFrameElement, timeout: number, urlNavigate: string): Promise<string> {
-        const contentWindow = iframe.contentWindow;
+    monitorIframeForHash(iframe: HTMLIFrameElement, timeout: number): Promise<string> {
         return new Promise((resolve, reject) => {
-            const maxTicks = timeout / BrowserConstants.POLL_INTERVAL_MS;
-            let ticks = 0;
+            /*
+             * Polling for iframes can be purely timing based,
+             * since we don't need to accout for interaction.
+             */
+            const nowMark = window.performance.now();
+            const timeoutMark = nowMark + timeout;
 
             const intervalId = setInterval(() => {
-                if (contentWindow.closed) {
-                    // Window is closed
-                    this.removeHiddenIframe(iframe);
```

Comment:
Why are we no longer removing the iframe?

Reply:
Ah, missed that, will fix.